### DISCUSSION TITLE
How are columns for a table defined in openapi connector?

### DISCUSSION THREAD

**Discussion by saurabh-shinde** [nineinchnick/trino-openapi#63] [Q&A]
I want to understand how table definition is done for every API. 
I see that every table in `default` schema is 'path' in `paths` in API spec. Want to get a better understanding on how table columns(other than API params) are determined.
I am trying to onboard Apache Atlas API as a catalog & I am seeing that not all 'keys' from API response are set as columns. 
TIA

**Comment by nineinchnick** [comment]
Please include the spec and open an issue if there are fields that are not mapped to columns.

https://github.com/nineinchnick/trino-openapi/issues/18 is about documenting the mapping of the API spec to the database schema.
